src/FirmwarePlugin/PX4/PX4FirmwarePlugin.cc

```diff
@@ -174,7 +174,7 @@ bool PX4FirmwarePlugin::setFlightMode(const QString& flightMode, uint8_t* base_m
 
 int PX4FirmwarePlugin::manualControlReservedButtonCount(void)
 {
-    return 8;   // 8 buttons reserved for rc switch simulation
+    return 0;   // 0 buttons reserved for rc switch simulation
```

Comment:
That's why I include https://github.com/mavlink/qgroundcontrol/pull/2332/files#diff-7179de35ff534916a12fd2e757566fe8R350. It doesn't set the button bits for unreserved buttons anymore. Effective PX4 stops trying to turn gamepad button presses into RC controller switch positions.

We have flight time on this and have not had issues with inadvertently getting an RC switch action from button presses.